style.css

```diff
@@ -77,4 +82,248 @@
 .icons-list li {
   display: inline-block;
   margin-right: 5%;
-}
\ No newline at end of file
+}
+
+/** here we start the second milestone **/
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100%, 1fr));
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-gap: 5%;
+  margin-top: 25%;
+  margin-bottom: 25%;
+}
+
+.middle {
+  display: flex;
+  flex-flow: column;
+  background-color: #fff;
+  border-radius: 1rem;
+}
+
+.middle-images {
+  order: -1;
+  align-self: center;
+  margin-top: 5%;
+}
+
+.middle-h1 {
+  font-style: normal;
+  font-weight: bolder;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0px;  */
+}
+
+.middle-paragraph {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 160%;
+  color: #344563;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0px;  */
+}
+
+.button-see-project {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 141%;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  color: #396df2;
+  border-color: #396df2;
+  border-radius: 0.5rem;
+  width: 125px;
+  height: 40px;
+  margin-top: 5%;
+  margin-bottom: 5%;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 10px; */
+}
```

Comment:
- [ ]Please remove commented code to make your code clean.

Reply:
Well resolved